explore/src/main/scala/explore/tabs/ObsTabContents.scala

```diff
@@ -66,13 +66,16 @@ final case class ObsTabContents(
 )(implicit val ctx: AppContextIO)
     extends ReactFnProps[ObsTabContents](ObsTabContents.component)
 
-object ObsTabTilesIds {
-  val NotesId: NonEmptyString         = "notes".refined
-  val TargetId: NonEmptyString        = "target".refined
-  val PlotId: NonEmptyString          = "elevationPlot".refined
-  val ConstraintsId: NonEmptyString   = "constraints".refined
-  val ConfigurationId: NonEmptyString = "configuration".refined
-}
+enum ObsTabTilesIds:
+  case NotesId, TargetId, PlotId, ConstraintsId, ConfigurationId, ItcId
+
+  def id: NonEmptyString = this match
+    case NotesId         => "notes".refined
+    case TargetId        => "target".refined
+    case PlotId          => "elevationPlot".refined
+    case ConstraintsId   => "constraints".refined
+    case ConfigurationId => "configuration".refined
+    case ItcId           => "itc".refined
```

Comment:
An alternative would be to use `enum ObsTabTilesIds(val id: NonEmptyString)` and skip the match here.